osu.Game/Screens/Select/PlaySongSelect.cs

```diff
@@ -20,6 +24,8 @@ public class PlaySongSelect : SongSelect
         private OsuScreen player;
         private readonly ModSelectOverlay modSelect;
         private readonly BeatmapDetailArea beatmapDetails;
+        private IEnumerable<Mod> originalMods;
+        private bool controlPressed;

```

Comment:
Just discussed with @peppy, let's modify `OnSelected()` to accept an input state parameter, which can be null when called through code.